### ISSUE TITLE
wrong weather template that does not report any forecast

### ISSUE THREAD

**Issue by DuduNord** [kalliope-project/kalliope_neuron_openweathermap#11]
hello.
I think that there is an error in the template example. OpenWeather does report some weather (listed in the debug mode) but the template is testing << tuesday['weather_status'] >> It should be  tuesday['07:00']['weather_status']

**Comment by Sispheor** [comment]
This is correct. Feel free to send a PR to update the doc.

**Comment by dataminer-x** [comment]
Actually, I can't get a response using this template, even with the "if" statement change. It pulls the data, but the message is empty. It looks like the times have changed in the data, for example, there is no '07:00' anymore, it's '08:00'. Even with those updates, if there's no data, it should report "I'm sorry, there is no forecast for..." but it just returns an empty message.

**Comment by Sispheor** [comment]
Yep. We haven't changed the doc. But I'm still wondering if the api change when summer/winter time.

**Comment by DuduNord** [comment]
maybe we can change the template to "read the available hour" and update the request..
